app/models/bookmark.rb

```diff
@@ -28,7 +28,7 @@ class Bookmark < ActiveRecord::Base
     where(conditions)
   }
 
-  scope :controller, lambda { |*args| where("controller = ?", (args.first || '')) }
+  scope :controller, ->(*args) { where("controller = ?", (args.first || '')) }
```

Comment:
I'm surprised this works on Ruby 1.9 as args are only supported from 2.0 I thought, but perhaps Rails is enhancing it.